src/components/UI/ReadMore/ReadMore.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+import styles from './ReadMore.module.scss';
+import {readMoreTxt} from './ReadMore.strings';
```

Comment:
We're not using this convention anymore.
Just create const var inside this file which defines the text.
`const READ_MORE_TXT = '....'`